src/test/java/net/sacredlabyrinth/phaed/simpleclans/managers/BungeeManagerTest.java

```diff
@@ -0,0 +1,29 @@
+package net.sacredlabyrinth.phaed.simpleclans.managers;
+
+import net.sacredlabyrinth.phaed.simpleclans.Clan;
+import net.sacredlabyrinth.phaed.simpleclans.ClanPlayer;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BungeeManagerTest {
+
+    @Test
+    public void updateFields() throws IllegalAccessException {
+        ClanPlayer cpOrigin = new ClanPlayer();
+        cpOrigin.setName("Test");
+        ClanPlayer cpDest = new ClanPlayer();
+        BungeeManager.updateFields(cpOrigin, cpDest);
+        Assert.assertEquals("Test", cpDest.getName());
+
+        Clan clanOrigin = new Clan();
+        clanOrigin.setMemberFee(0.222);
+        Clan clanDest = new Clan();
+        BungeeManager.updateFields(clanOrigin, clanDest);
+        Assert.assertEquals(0.222, clanDest.getMemberFee(), 0);
```

Comment:
Doesn't follow `Given-When-Then` tests style. 
Use `@Before` to create `cpOrigin`, `cpDest` and so on.